src/Core/Content/Category/Aggregate/CategoryTranslation/CategoryTranslationEntity.php

```diff
@@ -13,110 +13,41 @@ class CategoryTranslationEntity extends TranslationEntity
 {
     use EntityCustomFieldsTrait;
 
-    /**
-     * @var string
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
-     */
-    protected $categoryId;
+    protected string $categoryId;
 
-    /**
-     * @var string
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
-     */
-    protected $categoryVersionId;
+    protected string $categoryVersionId;
 
-    /**
-     * @var string|null
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
-     */
-    protected $name;
+    protected ?string $name;
```

Comment:
you should consider setting the default to `null`, as it could lead to errors, if the property is accessed, but not initialised.